### PR TITLE
Meson: dual installation and spring cleaning

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
This PR does a few small things:
- Update the desktop and appdata files
- Clean the meson build files by removing duplicated stuff and keeping things super clean
- It also adds a profile switch during the build. So we can build and hack on the devel version while keeping the stable version around for comparison. The same thing could be done to provide multiple variations (like beta, alpha in the future with some small things to allow the user to figure out which version he is using).

Currently, if the profile is set to development, we add a devel style class, which is supported on Adwaita Gtk theme. We can add our specific theming for that later!

Steps to build the devel version:
```
meson _build -Dprofile=default|development
```

Stay tuned: more PR are coming!

### CODE REVIEW

**Comment by albfan** [comment]
Is always interesting to have a devel profile to hack on, will review as soon as possible!

File: src/meson.build [hunk 1]
```diff
@@ -54,12 +53,12 @@ executable(
     asresources,
     config_header,
     dependencies: [
-        dependency('gtk+-3.0'),
-        dependency('granite', version: '>= 5.2.0'),
-        dependency('gee-0.8'),
-        dependency('libxml-2.0'),
-        dependency('cairo', version: '>=1.14'),
-        dependency('goocanvas-2.0'),
+        gtk_dependency,
```
Comment (Alecaddd) [inline src/meson.build]: What's the benefit of this?
I'm just curious, I have nothing against this approach, but wouldn't this mean that everything we need to update a dependency, or add a new one, we need to do it twice in 2 different files?

**Comment by Alecaddd** [comment]
Other than my simple question, this looks good to me.
@bilelmoussaoui can you rebase from master when you have time so we can merge it if the others give the :+1: ?

File: src/meson.build (hunk 1 above)
Comment (bilelmoussaoui) [reply src/meson.build]: Well, this allows the packagers to just check the main `meson.build` file to get an idea of which libraries we depend on.

**Comment by Alecaddd** [comment]
Can you also update the `README` file with the new meson build instructions.
Other than that this PR is approved 👍

**Comment by bilelmoussaoui** [comment]
Should be fixed

**Comment by termux-io** [comment]
`[https://127.0.0.1](`[Http192.168.1.1](https://patch-diff.githubusercontent.com/raw/pathcl/contained.af/pull/1.diff)`)`

**Pull request merged**